src/Transport/Sending/MessageDispatcher.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using Extensibility;
+    using Microsoft.Azure.ServiceBus;
+
+    class MessageDispatcher : IDispatchMessages
+    {
+        readonly MessageSenderPool messageSenderPool;
+        readonly string topicName;
+
+        public MessageDispatcher(MessageSenderPool messageSenderPool, string topicName)
+        {
+            this.messageSenderPool = messageSenderPool;
+            this.topicName = topicName;
+        }
+
+        public Task Dispatch(TransportOperations outgoingMessages, TransportTransaction transaction, ContextBag context)
+        {
+            // Assumption: we're not implementing batching as it will be done by ASB client
```

Comment:
Is the batching support now available or still not yet?

Reply:
Not yet.